<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>seans</class>
 <widget class="QDialog" name="seans">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Кінозал</string>
  </property>
  <property name="windowIcon">
   <iconset resource="movie_image.qrc">
    <normaloff>:/movie_image/logo_cinema.jpg</normaloff>:/movie_image/logo_cinema.jpg</iconset>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>541</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QWidget" name="gridWidget" native="true">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>481</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="horizontalSpacing">
       <number>6</number>
      </property>
      <item row="1" column="0">
       <widget class="QPushButton" name="s2_1">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="s1_1">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="s3_3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="s3_5">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="s2_3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="s1_6">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QPushButton" name="s2_6">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QPushButton" name="s3_6">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="s2_2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="s3_2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="s1_2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="s1_4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="s1_3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="s1_5">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="s3_4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="s3_1">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="s2_5">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="s2_4">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #fff; 
text-decoration: none; 
user-select: cursor;
background:rgb(255, 0, 0);
outline: none; 
border: 1px solid #e5c182;
border-radius:5px;
font-size:10px;
padding: 5px;</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhLowercaseOnly</set>
     </property>
     <property name="text">
      <string>Бронювати</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>300</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #fff; 
text-decoration: none; 
user-select: cursor;
background:rgb(0, 0, 0);
outline: none; 
border: 1px solid #e5c182;
border-radius:5px;
font-size:10px;
padding: 5px;</string>
     </property>
     <property name="text">
      <string>Купити</string>
     </property>
    </widget>
    <widget class="QPushButton" name="s4_2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>210</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="s5_2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>250</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="s4_3">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>210</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="s5_3">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>250</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="s4_4">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>210</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="s5_4">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>250</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="s4_1">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>210</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="s5_1">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>250</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>541</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:qlineargradient(spread:reflect, x1:0, y1:1, x2:1, y2:0, stop:0.0168539 rgba(0, 0, 0, 255), stop:0.955056 rgba(255, 255, 255, 255));
border: none;
font: 26pt &quot;Cambria&quot;;

</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>340</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #fff; 
text-decoration: none; 
user-select: cursor;
background:#14141a;
outline: none; 
border: 1px solid #e5c182;
border-radius:5px;
font-size:10px;</string>
     </property>
     <property name="text">
      <string>Закрити</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #fff; 
text-decoration: none; 
user-select: cursor;
background:#14141a;
outline: none; 
border: 1px solid #e5c182;
border-radius:5px</string>
     </property>
     <property name="text">
      <string>1 ряд</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #fff; 
text-decoration: none; 
user-select: cursor;
background:#14141a;
outline: none; 
border: 1px solid #e5c182;
border-radius:5px</string>
     </property>
     <property name="text">
      <string>2 ряд</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #fff; 
text-decoration: none; 
user-select: cursor;
background:#14141a;
outline: none; 
border: 1px solid #e5c182;
border-radius:5px</string>
     </property>
     <property name="text">
      <string>3 ряд</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #fff; 
text-decoration: none; 
user-select: cursor;
background:#14141a;
outline: none; 
border: 1px solid #e5c182;
border-radius:5px</string>
     </property>
     <property name="text">
      <string>VIP</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #fff; 
text-decoration: none; 
user-select: cursor;
background:#14141a;
outline: none; 
border: 1px solid #e5c182;
border-radius:5px;
font-size:10px;</string>
     </property>
     <property name="text">
      <string>VIP</string>
     </property>
    </widget>
    <widget class="QLabel" name="l_vil">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>300</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0); 
text-decoration: none; 
user-select: cursor;
background:rgb(255, 255, 255);
outline: none; 
border: 1px solid #e5c182;
border-radius:5px;
font-size:10px;</string>
     </property>
     <property name="text">
      <string>Вільно</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <zorder>gridWidget</zorder>
    <zorder>radioButton</zorder>
    <zorder>s4_2</zorder>
    <zorder>s5_2</zorder>
    <zorder>s4_3</zorder>
    <zorder>s5_3</zorder>
    <zorder>s4_4</zorder>
    <zorder>s5_4</zorder>
    <zorder>s4_1</zorder>
    <zorder>s5_1</zorder>
    <zorder>label_2</zorder>
    <zorder>pushButton</zorder>
    <zorder>label_3</zorder>
    <zorder>label_4</zorder>
    <zorder>label_5</zorder>
    <zorder>label_6</zorder>
    <zorder>label_7</zorder>
    <zorder>l_vil</zorder>
    <zorder>radioButton_2</zorder>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
 </widget>
 <resources>
  <include location="movie_image.qrc"/>
 </resources>
 <connections/>
</ui>
